Install.md

```diff
@@ -66,7 +66,12 @@ If you can't use any of the package managers, you can also download [git-credent
     ```
     git-credential-manager install
     ```
-
+7. If the last command failed because Java couldn't be found make sure you have the `JAVA_HOME` variable set correctly in your environment.
+    1. Run `which java` to find out where Java is installed, this will show e.g. `/usr/bin/java`.
+    2. Run `echo $JAVA_HOME` to find out if JAVA_HOME is set, in our example case this should be `/ùsr`.
+    3. Edit your environment variables by running `sudo -H gedit /etc/environment` and adding (or replacing) the line `JAVA_HOME=/usr`.

```

Comment:
Would it be possible to offer a user specific solution to setting `JAVA_HOME` here to avoid specifying a sudo command?

Reply:
Hi novak,

using bash you could add it to your `.bash_profile`

```suggestion
    3. Edit your environment variables by running `sudo -H gedit /etc/environment` and adding (or replacing) the line `JAVA_HOME=/usr` or for `bash` by adding `export JAVA_HOME=/usr` to your `~/.profile`.
```